packages/loopring_v3/contracts/amm/libamm/AmmData.sol

```diff
@@ -112,5 +112,6 @@ library AmmData
 
         // A map from a user to the forced exit.
         mapping (address => PoolExit) forcedExit;
+        mapping (bytes32 => uint) approvedTx;
```

Comment:
Not that it matters I think, but I guess this could just be `=> bool` because we don't actually use the timestamp now.

Reply:
changed to use bool